automation/scripts/auto-deploy.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
```

Comment:
I removed this script from the monorepo originally because it sort of scares me but if you have been using it successfully then ✅

Reply:
tbh i've never used this script, i think @deepthiskumar added this back in?

Reply:
I did use it in the beginning but later switched to simply doing a `terraform apply` and `destroy`. You can delete if it is not being used anymore